### ISSUE TITLE
Vite support

### ISSUE THREAD

**Issue by jiridenteo** [JuniorTour/vue-template-babel-compiler#46]
Hi,
how can I install this great package for Vite? Is it compatible? Thanks

**Comment by JuniorTour** [comment]
Yes, it works for vite!

### 1. With `vitejs/vite-plugin-vue2` plugin (https://github.com/vitejs/vite-plugin-vue2)

Simply add options like below:

``` js
// vite.config.js
import { defineConfig } from "vite";
import { createVuePlugin } from "vite-plugin-vue2";

export default defineConfig({
  plugins: [
    createVuePlugin({
      jsx: true,
      vueTemplateOptions: {
        compiler: require('vue-template-babel-compiler'),
      },
    }),
  ],
}); 
```

### 2. `underfin/vite-plugin-vue2` plugin (https://www.npmjs.com/package/vite-plugin-vue2)

This plugin contain `vue-template-babel-compiler@1.2.0` out of box: https://github.com/underfin/vite-plugin-vue2/blob/master/package.json#L71

Just add `vite-plugin-vue2` plugin, new syntax like `optional chaining` will work.

**Comment by stephane303** [comment]
Is it possible to use this with vite-plugin-vue2 ? https://github.com/vitejs/vite-plugin-vue2

**Comment by JuniorTour** [comment]
@stephane303 sure, see reply above you, it contains detail usage.

**Comment by stephane303** [comment]
Yes thanks, I got confused as yes it works with 2.7.14, but not (yet) with jest ... but I will get there ....
